res/database/Default/user/therob.xml

```diff
@@ -2035,69 +2123,67 @@
 				<member index="0" function="1">9104f9c1-7a0f-4bc0-a34c-389ce282eebf</member>
 				<member index="1" function="8">3d37e02e-78f2-4749-9de5-429e31b5590b</member>
 			</staff>
-			<groups target_groups="0" pro_pressure_groups="0" contra_pressure_groups="0" />
-			<data country="D" distribution="2" maingenre="204" subgenre="" flags="8" blocks="4" price_mod="1.00" />
-			<releaseTime year_relative="-5" />
-			<ratings critics="89" speed="38" />
+			<groups target_groups="6" pro_pressure_groups="0" contra_pressure_groups="0" />
+			<data country="D" distribution="2" maingenre="204" subgenre="103" flags="9" blocks="4" broadcast_flags="512" price_mod="0.25" />
```

Comment:
the price modifier looks odd - relatively good rating, live, no broadcast limit...